### ISSUE TITLE
Standarized Viewport Code

### ISSUE THREAD

**Issue by cslotboom** [u-anurag/OpenSeesPy#43]
@u-anurag I was thinking of replacing the viewpoint setting code in **plot_modeshape**, **plot_model**, **plot_deformedshape** with the existing **_setStandardViewport** function.

We can cut down on the lines of code this way. If you are okay with it I'll roll this out in the next pull request.

For example:
```
nodeMins = np.array([min(DeflectedNodeCoordArray[:,0]),min(DeflectedNodeCoordArray[:,1]),min(DeflectedNodeCoordArray[:,2])])
nodeMaxs = np.array([max(DeflectedNodeCoordArray[:,0]),max(DeflectedNodeCoordArray[:,1]),max(DeflectedNodeCoordArray[:,2])])
				
xViewCenter = (nodeMins[0]+nodeMaxs[0])/2
yViewCenter = (nodeMins[1]+nodeMaxs[1])/2
zViewCenter = (nodeMins[2]+nodeMaxs[2])/2
		
view_range = max(max(DeflectedNodeCoordArray[:,0])-min(DeflectedNodeCoordArray[:,0]), 
					max(DeflectedNodeCoordArray[:,1])-min(DeflectedNodeCoordArray[:,1]), 
					max(DeflectedNodeCoordArray[:,2])-min(DeflectedNodeCoordArray[:,2]))

ax.set_xlim(xViewCenter-(view_range/2), xViewCenter+(view_range/2))
ax.set_ylim(yViewCenter-(view_range/2), yViewCenter+(view_range/2))
ax.set_zlim(zViewCenter-(view_range/2), zViewCenter+(view_range/2))
```

Would be replaced with:

`_setStandardViewport(fig, ax, DeflectedNodeCoordArray, ndm)`

**Comment by u-anurag** [comment]
Sure! Make sure you run the examples with this.

**Comment by cslotboom** [comment]
Sounds good. Nothing will get committed until all the examples work!

**Comment by cslotboom** [comment]
Addressed in #47.
